app/views/devise/registrations/start.html.slim

```diff
@@ -1,11 +1,12 @@
 - title t('titles.registrations.start')
 
-h1.h3.my0 = t('headings.create_account')
+h1.h3.my0 = t("headings.create_account_#{service_provider}", sp: @sp_name)
```

Comment:
The problem is we can never predict the future, but we can design the code so that it is easy to change. As Sandi Metz says in her book *Practical Object-Oriented Design in Ruby*:

> Writing Code That Embraces Change
> You can arrange the code so that Gear will be easy to change even if you don’t know what changes will come. Because change is inevitable, coding in a changeable style has big future payoffs. As an additional bonus, coding in these styles will improve your code, today, at no extra cost.